rfcs/icon-component.md

```diff
@@ -48,7 +48,7 @@ From left-to-right based on the image above:
 ## `Icon` API
 
 * `size: number|string` - Optional, Defaults to `scale500` (12px)
-  Size of element, will be passed to the svg width/height style. Can also be a baseui `theme.sizing` constant like `sizing500`.
+  Size of element, will be passed to the svg width/height style. Can also be a baseui `theme.sizing` constant like `scale500`.
```

Comment:
i know it is not part of this PR, but should we move the usage example up, so it follows the order we have in the rest of the rfcs?

Reply:
also, should we include an example with buttons too?

Reply:
Good call, will add.

Reply:
Wow I really like how the icons looks in your [live preview](https://deploy-preview-269--baseui.netlify.com/?knob-Size=48px&knob-Color=0&knob-Button%20Kind=primary&selectedKind=Icon&selectedStory=Icons%20in%20Button&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel).

Maybe it's just me but the start and end enhancers in the button seems to have too much whitespace. What do you think @nguyensomniac ?